bindings/python/tools/python/ostk/astrodynamics/display.py

```
@@ -121,36 +129,26 @@ def __init__(
         interval: RealInterval,
         trajectory_step: Duration,
         access_step: Duration,
-        ground_station_lla: LLA,
-        color: str,
+        ground_station_lla: LLA | None = None,
+        color: str | None = None,
     ):
+        if ground_station_lla is not None and color is not None:
+            warnings.warn(
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)